### PR TITLE
Documentation updates: unload into s3

### PR DESCRIPTION
**Pull request by soyeric128**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Changelog

- Documentation

Added the command COPY INTO <location>

## Related Issues

Fixes #issue

### CODE REVIEW

File: docs/doc/30-reference/30-sql/10-dml/dml-copy-into-location.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+| ----------- | ----------- | --- |
+| `SIZE_LIMIT = <num>` | Number (> 0) that specifies the maximum rows of data to be unloaded for a given COPY statement. Default `0` | Optional |
+
+## Examples
+
+
+### Unloading into an Internal Stage
+
+```sql
+//Create a table
```
Comment (BohuTANG) [inline docs/doc/30-reference/30-sql/10-dml/dml-copy-into-location.md:86]:
```suggestion
-- Create a table
```

File: docs/doc/30-reference/30-sql/10-dml/dml-copy-into-location.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+CREATE TABLE test_table (
+     id INTEGER,
+     name VARCHAR,
+     age INT
+);
+
+//Insert data into the table
```
Comment (BohuTANG) [inline docs/doc/30-reference/30-sql/10-dml/dml-copy-into-location.md:93]:
```suggestion
-- Insert data into the table
```

File: docs/doc/30-reference/30-sql/10-dml/dml-copy-into-location.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+insert into test_table (id,name,age) values(1,'2',3), (4, '5', 6);"
+
+//Create an internal stage
```
Comment (BohuTANG) [inline docs/doc/30-reference/30-sql/10-dml/dml-copy-into-location.md:96]:
```suggestion
-- Create an internal stage
```

File: docs/doc/30-reference/30-sql/10-dml/dml-copy-into-location.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+CREATE STAGE s2;
+
+
+//Unload the data in the table into the stage as a CSV file
```
Comment (BohuTANG) [inline docs/doc/30-reference/30-sql/10-dml/dml-copy-into-location.md:100]:
```suggestion
-- Unload the data in the table into the stage as a CSV file
```

File: docs/doc/30-reference/30-sql/10-dml/dml-copy-into-location.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+copy into @s2 from test_table FILE_FORMAT = (type = 'CSV');
+
+//Unload the data from a query into the stage as a Parquet file
```
Comment (BohuTANG) [inline docs/doc/30-reference/30-sql/10-dml/dml-copy-into-location.md:103]:
```suggestion
-- Unload the data from a query into the stage as a Parquet file
```

**Comment by BohuTANG** [comment]
Usually, we choose one as the `Changelog`, for example, `Documention`.
This will show in the release for the category:
image

**Comment by soyeric128** [comment]
> Usually, we choose one as the `Changelog`, for example, `Documention`.
> This will show in the release for the category:
> image

Thanks, I committed the suggested changes and updated the changelog.

**Comment by drmingdrmer** [comment]
> Usually, we choose one as the `Changelog`, for example, `Documention`. This will show in the release for the category: 

Where should a `Documentation` keyword be to let the bot read it?
Commit title, commit body, PR title, PR body?

**Pull request merged**